src/xplugd.h

```diff
@@ -42,7 +42,7 @@
 #include <X11/extensions/Xrandr.h>
 
 #define MSG_LEN 128
-#define XPLUGRC "~/.xplugrc"
+#define XPLUGRC "~/.config/xplugrc"
```

Comment:
I'm OK with respecting XDG, but the fallback should not assume the `~/.config` directory exists.  Please revert this.

Reply:
> If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

This is what the spec says.

Reply:
Regardless of what a spec. says, not all users of xplugd follow the spec, or have the necessary tools set up, so the fallback should not assume the ~/.config directory exists. 

The code above looks for the environment variable, if it exists we use that, if not we should default to the old behavior also for the sake of compatibility with previous xplugd releases. 

Sorry, but I'm a bit astounded by your comment. You are proposing a change to the default behavior of xplugd and the maintainer (I) ask you kindly to at least preserve the fallback.  Why not just do that?

Reply:
Because the purpose of this PR is to follow the spec. Sorry if I came across as being rude. If you do not want to follow the spec, I'll kindly close the PR.

Reply:
Specs. are generally a good thing, and when building a new application it's very useful to follow them to the letter. For existing applications it's always a fine line to tread between usability, standards, and backwards compatibility.

Like I said, the first part of the PR is good (apart from the strcat() thing), since it checks for the XDG environment variable yet has fallback handling.  We just don't agree on what the fallback should be.